src/providers/mbed_crypto/key_management.rs

```diff
@@ -41,6 +41,18 @@ pub fn get_key_id(
     }
 }
 
+/// Get key Attributes from the Key Info Manager.
+pub fn get_key_attributes(
```

Comment:
Not sure why we didn't have this to begin with, I think we cache attributes separately in the Mbed Crypto provider

Reply:
I think that is because Mbed Crypto handle the key attributes separately, we don't need to do it ourselves.